app/src/main/java/com/kickstarter/ApplicationModule.java

```diff
@@ -406,20 +415,20 @@ Application provideApplication() {
     return this.application;
   }
 
-  @Provides
-  @KoalaTracker
-  @Singleton
-  static Koala provideKoala(final @ApplicationContext @NonNull Context context, final @NonNull CurrentUserType currentUser,
-    final @NonNull Build build, final @NonNull CurrentConfigType currentConfig, final @NonNull ExperimentsClientType experimentsClientType) {
-    return new Koala(new KoalaTrackingClient(context, currentUser, build, currentConfig, experimentsClientType));
-  }
-
   @Provides
   @LakeTracker
   @Singleton
-  static Koala provideLake(final @ApplicationContext @NonNull Context context, final @NonNull CurrentUserType currentUser,
-    final @NonNull Build build, final @NonNull CurrentConfigType currentConfig, final @NonNull ExperimentsClientType experimentsClientType) {
-    return new Koala(new LakeTrackingClient(context, currentUser, build, currentConfig, experimentsClientType));
+  static AnalyticEvents provideAnalytics(
+          final @ApplicationContext @NonNull Context context,
+          final @NonNull CurrentUserType currentUser,
+          final @NonNull Build build,
+          final @NonNull CurrentConfigType currentConfig,
+          final @NonNull ExperimentsClientType experimentsClientType,
+          final @Nullable Analytics segment) {
+    final LakeTrackingClient lakeTrackingClient = new LakeTrackingClient(context, currentUser, build, currentConfig, experimentsClientType);
```

Comment:
Is there a reason that you are creating instances of these tracking clients inside of this provider method? I thought it would make more sense to have these dependencies handled by their own dagger provider methods in this ApplicationModule.